config/parameters.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $container): void {
+    $container->parameters()->set('ignore_uncovered_internal_classes', false);
+    $container->parameters()->set('supported_analyser_types', ['class', 'use']);
+    $container->parameters()->set('baseline', null);
+    $container->parameters()->set('paths', []);
+    $container->parameters()->set('exclude_files', []);
+    $container->parameters()->set('layers', []);
+    $container->parameters()->set('ruleset', []);
+    $container->parameters()->set('skip_violations', []);
+
+    $container->parameters()->set('formatter_graphviz.groups', []);
```

Comment:
I am not an expert in Symfony so I am just wondering why do we need the nesting for `formatter_graphviz.*` but not for the rest?